app/src/org/commcare/android/resource/installers/XFormAndroidInstaller.java

```diff
@@ -70,6 +70,7 @@ public XFormAndroidInstaller(String localLocation, String localDestination, Stri
     public boolean initialize(AndroidCommCarePlatform platform, boolean isUpgrade) throws
             IOException, InvalidReferenceException, InvalidStructureException,
             XmlPullParserException, UnfullfilledRequirementsException {
+        super.initialize(platform, isUpgrade);

```

Comment:
Interesting - OOC why were we able to get away with *not* calling `super` for so long but we need to now?

Reply:
Super class `FileSystemInstaller` earlier had `initialize` as an `abstract` method and I just  converted it to a `non abstract` one in this PR to be able to add common functionality to check for missing files before we proceed.